src/test/java/com/qaprosoft/carina/demo/phpTravels/LoginPageTest.java

```diff
@@ -25,68 +26,64 @@ public void testForgotAccButton() {
 
     @Test
     public void testLoginAdminAcc() {
-        loginIn("admin@phptravels.com", "demoadmin");
+        login();
         DashboardPage dashboardPage = new DashboardPage(getDriver());
         Assert.assertTrue(dashboardPage.isPageOpened());
     }
 
     @Test
     public void testChangeName() {
-        loginIn("admin@phptravels.com", "demoadmin");
+        login();
         DashboardPage dashboardPage = new DashboardPage(getDriver());
-        LeftMenuBar leftMenu =dashboardPage.getLeftMenuBar();
+        SettingsPage settingsPage = dashboardPage.getLeftMenuBar().openSettingsSubmenu();
 
-        SettingsPage settingsPage = leftMenu.openSettingsSubmenu();
         settingsPage.typeNameText("JS Travels | Travel Technology Partner");
         settingsPage.clickSaveChangeButton();
-        Assert.assertTrue(settingsPage.getChangesSavedText());
+
+        Assert.assertTrue(settingsPage.getChangesSaved());
     }
 
     @Test
     public void testUnpaidBookingsMenu() {
-        loginIn("admin@phptravels.com", "demoadmin");
+        login();
         DashboardPage dashboardPage = new DashboardPage(getDriver());
-        NavigationBar navigationBar = dashboardPage.getNavigationBar();
-
-        BookingsPage bookingsPage = navigationBar.openBookingsPage();
-        BookingsMenu bookingsMenu = bookingsPage.getBookingsButtons();
+        BookingsMenu bookingsMenu = dashboardPage.getNavigationBar().openBookingsPage().getBookingsMenu();
+        BookingsPage bookingsPage = new BookingsPage(getDriver());
         bookingsMenu.clickUnpaidBookings();
-        Table tableChosen = bookingsPage.getBookingsTable();
-        String bookingStatus = tableChosen.getChosenPaidStatus("1").toUpperCase();
+        String bookingStatus = bookingsPage.getBookingsTable().getChosenPaidStatus("1").toUpperCase();
         Assert.assertEquals(bookingStatus, "UNPAID");
     }
 
     @Test
     public void testDeleteBooking() {
-        loginIn("admin@phptravels.com", "demoadmin");
+        login();
         DashboardPage dashboardPage = new DashboardPage(getDriver());
         NavigationBar navigationBar = dashboardPage.getNavigationBar();
 
         BookingsPage bookingsPage = navigationBar.openBookingsPage();
-        BookingsMenu bookingsMenu = bookingsPage.getBookingsButtons();
+        BookingsMenu bookingsMenu = bookingsPage.getBookingsMenu();
         bookingsMenu.clickUnpaidBookings();
         Table tableChosen = bookingsPage.getBookingsTable();
         tableChosen.deleteBooking("1");
-        Assert.assertTrue(tableChosen.isAnyElementPresent());
+        getDriver().switchTo().alert();
```

Comment:
please read this helpful articles how to work work alerts
https://www.guru99.com/alert-popup-handling-selenium.html